src/app/fda/application/model/application.model.ts

```diff
@@ -85,23 +172,15 @@ export interface ApplicationIngredient {
   modifyDate?: number;
   farmSubstanceId?: number;
   farmSubstance?: string;
-  ingredientName?: string;
-  substanceId?: string;
+  _substanceUuid?: string;
+  _approvalID?: string;
+  _name?: string;
   activeMoietyName?: string;
   activeMoietyUnii?: string;
   subRelationshipList?: Array<SubRelationship>;
 }
```

Comment:
There is a property that seems to be missing here according to the compiler that I had to add manually `ERROR in src/app/fda/application/application-form/ingredient-form/ingredient-form.component.html(65,16): Property 'substanceCode' does not exist on type 'ApplicationIngredient'. Did you mean 'substanceKey'? 


`